test/component/Label.spec.tsx

```diff
@@ -143,17 +151,15 @@ describe('<Label />', () => {
   });
 
   it('Renders label by label props with animation enabled', () => {
-    jest.useFakeTimers();
-
     const { container } = render(
       <LineChart width={400} height={400} data={data} margin={{ top: 20, right: 20, bottom: 20, left: 20 }}>
         <Line type="monotone" dataKey="uv" stroke="#ff7300" label={{ value: 'hello', position: 'center' }} />
       </LineChart>,
     );
-    // run all timers so animation finishes and labels render
-    jest.runAllTimers();
 
-    expect(container.querySelectorAll('.recharts-line .recharts-line-curve').length).toEqual(1);
-    expect(screen.getByText(/400/i)).toBeInTheDocument();
+    setTimeout(() => {
+      expect(container.querySelectorAll('.recharts-line .recharts-line-curve').length).toEqual(1);
+      expect(screen.getByText(/400/i)).toBeInTheDocument();
```

Comment:
Fake timers weren't working, but using a regular timeout seemed to resolve the issue.

Reply:
I spoke too soon :/ I incorrectly thought this fixed the tests, but using an actual timer meant the expect was never called as the test was no longer in scope. There are some weird issues with timers and whatnot with Vitest that I am doing my best to figure out, but for now, my boneheaded approach of using real timers isn't an option 🫠